src/test/resources/features/99d_employment_qa_smoke.feature

```diff
@@ -0,0 +1,153 @@
+@mvp @ZAP @employment
```

Comment:
this should only have a @smoke tag because its ran as part of the smoke tests and not mvp/zap/employment